app/src/main/java/com/example/rocketreserver/LaunchListFragment.kt

```diff
@@ -62,18 +61,20 @@ class LaunchListFragment : Fragment() {
                 }
 
                 cursor = response.data?.launches?.cursor
+
                 if (response.data?.launches?.hasMore != true) {
                     break
                 }
             }
 
             adapter.onEndOfListReached = null
+
             channel.close()
         }
 
-        adapter.onItemClicked = { launch ->
+        adapter.onItemClicked = {
```

Comment:
Same as above, do you mind either keeping the variable name or updating the doc?